settings.py

```diff
@@ -5,13 +5,4 @@
 # https://www.openmicroscopy.org/
 #   site/support/omero5.2/developers/Web/CreateApp.html
 
-
-def identity(x):
-    return x
-
-
-# import json
-CUSTOM_SETTINGS_MAPPINGS = {
-    "omero.web.figure.version":
-    ["OMERO_FIGURE_VERSION", "1.2.1", identity, None]
-}
+OMERO_FIGURE_VERSION = "1.2.1_dev"
```

Comment:
I guess the only concern would be if someone unwittingly removed the ```package.json``` from the dist directory. But I guess we can assume they won't do that.